api/app/controllers/purchase_lists_controller.rb

```diff
@@ -2,61 +2,110 @@ class PurchaseListsController < ApplicationController
   before_action :set_purchase_list, only: [:show, :update, :destroy]
 
   # GET /purchase_lists
-  # GET /purchase_lists.json
   def index
     @purchase_lists = PurchaseList.all
     render json: fmt(ok, @purchase_lists)
   end
 
   # GET /purchase_lists/1
-  # GET /purchase_lists/1.json
   def show
     render json: fmt(ok, @purchase_list)
   end
 
   # POST /purchase_lists
-  # POST /purchase_lists.json
   def create
-    @purchase_list = PurchaseList.create(purchase_list_params)
-    render json: fmt(created, @purchase_list)
+    @purchase_list = PurchaseList.new(purchase_list_params)
+
+    if @purchase_list.save
+      render json: fmt(created, @purchase_list)
+    else
+      render json: fmt(unprocessable_entity, [], @purchase_list.errors.full_messages.join(', ')), status: :unprocessable_entity
```

Comment:
```suggestion
      render json: fmt(unprocessable_entity, [], @purchase_list.errors.full_messages.join(', '))
```
fmtの中にstatusもあるからいらない